.gitignore

```diff
@@ -32,5 +32,14 @@ __pycache__/
 /frontend-dist/
 /scratch/
 /packages-dist/
-/.coverage
+.coverage
 /users.db
+
+# pdm
+.pdm.toml
+.pdm-python
+.pdm-build/
+__pypackages__/
```

Comment:
I don't think we need this given that we're not using pdm

Reply:
ah true, leftovers from when i tried first with pdm. will remove them.